cirq-core/cirq/experiments/two_qubit_xeb.py

```diff
@@ -373,6 +375,7 @@ def parallel_xeb_workflow(
         random_state: The random state to use.
         ax: the plt.Axes to plot the device layout on. If not given,
             no plot is created.
```

Comment:
Can we have this return an object with a `plot` method instead of requiring the user to pass `ax` here? You might not know that you want to plot it until after you take the data.

Reply:
done